src/ar.erl

```diff
@@ -202,6 +206,10 @@ parse(["enable", Feature | Rest ], O = #opts { enable = Enabled }) ->
 	parse(Rest, O#opts { enable = [ list_to_atom(Feature) | Enabled ] });
 parse(["disable", Feature | Rest ], O = #opts { disable = Disabled }) ->
 	parse(Rest, O#opts { disable = [ list_to_atom(Feature) | Disabled ] });
+parse(["gateway", Port, Domain | Rest ], O = #opts { gateway = off }) ->
+	parse(Rest, O#opts { gateway = {on, Port, Domain} });
```

Comment:
The `on` atom looks unnecessary to me, but if you see value in it, then I don't mind keeping it.